upi/openstack/update_ign.py

```diff
@@ -130,60 +130,73 @@ def update(hostname,ignition):
     config_data = {}
 
     ifcfg_ens3 = ("""TYPE=Ethernet
-    DEVICE=""" + node_interface + """
-    ONBOOT=yes
-    BOOTPROTO=dhcp
-    DEFROUTE=yes
-    PROXY_METHOD=none
-    BROWSER_ONLY=no
-    MTU=""" + neutron_network_mtu + """
-    IPV4_FAILURE_FATAL=no
-    IPV6INIT=no""").encode()
+DEVICE=""" + node_interface + """
+ONBOOT=yes
+BOOTPROTO=dhcp
+DEFROUTE=yes
+PROXY_METHOD=none
+BROWSER_ONLY=no
+MTU=""" + neutron_network_mtu + """
+IPV4_FAILURE_FATAL=no
+IPV6INIT=no
+ETHTOOL_OPTS="-K ens3 tx-checksum-ip-generic off"
+NAME="System ens3"
+UUID=21d47e65-8523-1a06-af22-6f121086f085
```

Comment:
Here, and also below in ens4, we are hard-coding the UUID. Do we know that its okay for the UUID to not be unique across hosts?

Its perhaps less problematic in the case of the sub-interface (the third one below).

Reply:
I doubt there will be any issues from a networking perspective (the network protocols are not aware of this UUID). But if the OS (or any Network Manager that is used by the OS) uses this UUID to derive another value (I vaguely recall reading somewhere that storage LUNs are calculated using this), then it might be problematic.

Reply:
for the rhcos machines, the name ens3 and ens4 is fixed across all OSP versions. The above update_ign.py is also tested in Yulia's setup, which worked fine. To verify it further, I can try a couple of times fresh install too, to ensure that MCO does not complain about the difference. And regarding the networking side, yes, this will not create any issue, however, for the latter part I believe, this will require to debug further that will the static UUID works.

Reply:
My point is not about about the interface names, its about the static UUID we will end up using. Also my worry is that merely installing it the way we do may not exercise all aspects of the installation (like on the storage side). Perhaps we can check with the RH team if its okay to hardcode these UUIDs?

Reply:
Absolutely, you are right. I can ask the Redhat Team on slack, for the same and how we can add expected UUIDs. Lets, hold this PR then until we get all of the answers.